libs/watt/src/lib/components/expandable-card/watt-expandable-card.component.ts

```diff
@@ -55,10 +53,7 @@ export class WattExpandableCardComponent {
   _content?: WattExpandableCardContentDirective;
   /** Whether the card is elevated or has solid border */
   @Input() variant: 'solid' | 'elevation' = 'elevation';
-  @HostBinding('class')
-  get cssClass() {
-    return `watt-expandable-card watt-${this.variant}`;
-  }
+  variantClass = `watt-${this.variant}`;
```

Comment:
issue: the value of `variantClass` is not updated once the component is initialized. So the class stays `watt-elevation` even if the Input is `solid`.

Using `class="watt-expandable-card watt-{{ variant }}"` should work fine.